content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2864,13 +2864,16 @@ In [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and later, Pri
 
 In the following example, `tags` is ignored because it refers to an invalid model (no `@id` on `Product`):
 
+<!-- prettier-ignore-start -->
+
```

Comment:
```suggestion
<!-- prettier-ignore-start -->
```

Reply:
@janpio that was a very silly mistake 😬 Fixing it now